neuralcompression/functional/complexity.py

```diff
@@ -0,0 +1,170 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+"""
+
+from copy import copy
+from typing import Any, Dict, Optional, Sequence, Tuple
+
+from fvcore.nn import FlopCountAnalysis
+from fvcore.nn.jit_handles import Handle, generic_activation_jit
+from torch import nn
+
+# These ops aren't supported (i.e. are ignored) by fvcore - we calculate
+# their flops by counting 1 flop per element of the output tensor. Note that
+# this approach handles broadcasting input shapes.
+_SINGLE_FLOP_OPS_TO_ADD = [
+    f"aten::{name}"
+    for name in [
+        "abs",
+        "add",
+        "div",
+        "mul",
+        "neg",
+        "rsub",
+        "sign",
+        "sub",
```

Comment:
This becomes slightly tricky based on https://github.com/facebookresearch/NeuralCompression/pull/6/files#r680301814

Basically, you'll count multiply adds as 2x the FLOPs based on this. I'm not sure what we should be doing for this. On one hand, this will generally contribute a small number to the FLOPs, so it'll probably not matter.

WDYT @mmuckley ?